packages/mask/src/components/InjectedComponents/ProfileTabContent.tsx

```diff
@@ -178,6 +180,12 @@ export function ProfileTabContent(props: ProfileTabContentProps) {
         })
     }, [retryIdentity])
 
+    useEffect(() => {
+        return MaskMessages.events.NFTProjectTwitterDetect.on((ev) => {
```

Comment:
Please don't make it depend on a remote event. Can we receive sth as a prop and derive it into an address?